POSserver/server/models/contractor.py

```diff
@@ -80,7 +82,7 @@ class Contractor(models.Model):
         ("VI", "Virgin Islands"),
         ("GU", "Guam"),
     )
-    state = models.CharField(max_length=2, choices=state_choices, default="Alabama")
+    state = models.CharField(max_length=50, choices=state_choices, default="Alabama")
     zipcode = models.CharField(max_length=10, null=False, blank=False)
     business_name = models.CharField(max_length=100, null=False, blank=False)
     # password = models.CharField(max_length=50, null=False, blank=False)
```

Comment:
is business_name optional?

Reply:
I believe we have since changed it to be optional with the idea that not all self-employed persons or persons who contract self-employed persons have business names.